python/ngen_cal/tests/test_configuration.py

```diff
@@ -1,30 +0,0 @@
-import pytest
-from typing import TYPE_CHECKING, Generator
-
-from .utils import config
-
-if TYPE_CHECKING:
-    from ngen.cal.configuration import Configuration
-"""
-    Test suite for reading and manipulating ngen configration files
-"""
-
-# TODO determmine if any unit tests of General config are appropriate
-# pydantic is thoroughly tested upstream, but it may be a good idea to
-# write some simple tests around the defaults
-# may also want to implment some testing around the Model union/parsing
-
-# @pytest.mark.usefixtures("conf", "realization_config")
-# def test_config_file(conf: 'Configuration', realization_config: str) -> None:
-#     """
-#         Test configuration property `config_file`
-#     """
-#     assert conf.config_file == realization_config
-
-# @pytest.mark.usefixtures("conf")
-# def test_catchments(conf: 'Configuration') -> None:
-#     """
-#         Ensure that only the catchment marked with "calibration" is used in the configuration
```

Comment:
I don't recall why these tests were all commented, but do you see any value in reviving any of them or some form of them?

Reply:
Probably, Ill open an issue to track re-adding them. For now, lets axe dead code though.

Reply:
See https://github.com/NOAA-OWP/ngen-cal/issues/165